src/shared/ipvlan-util.c

```diff
@@ -0,0 +1,23 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <net/if.h>
+
+#include "conf-parser.h"
```

Comment:
This is not necessary.

Reply:
`DEFINE_STRING_TABLE_LOOKUP` ?

Reply:
@yuwata is right, this include is not necessary. string-table.h makes DEFINE_STRING_TABLE_LOOKUP available, and that's all we need